lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -131,37 +132,23 @@ def queue_job(self, job_wrapper):
             self.__get_k8s_job_spec(ajs)
         )
 
-        # Checks if job exists and is trusted, or if it needs re-creation.
         job = Job(self._pykube_api, k8s_job_obj)
-        job_exists = job.exists()
-        if job_exists and not self._galaxy_instance_id:
```

Comment:
This piece of code is related to Galaxy instances being spin up again on k8s clusters where they were running in the past or where another Galaxy instance might have run. Could you explain why you are removing this?

Reply:
I see, if you use the internal cluster UUID then this is never really a problem.